compiler/rustc_middle/src/ty/flags.rs

```diff
@@ -137,7 +137,9 @@ impl FlagComputation {
             &ty::Infer(infer) => {
                 self.add_flags(TypeFlags::STILL_FURTHER_SPECIALIZABLE);
                 match infer {
-                    ty::FreshTy(_) | ty::FreshIntTy(_) | ty::FreshFloatTy(_) => {}
+                    ty::FreshTy(_) | ty::FreshIntTy(_) | ty::FreshFloatTy(_) => {
```

Comment:
probably want to do the same thing for `ConstKind::Infer(InferConst::Fresh(..))`?